scripts/zones/La_Theine_Plateau/npcs/Vicorpasse.lua

```diff
@@ -27,7 +27,7 @@ function onTrigger(player,npc)
 	if(player:getCurrentMission(SANDORIA) == THE_RESCUE_DRILL) then
 		local MissionStatus = player:getVar("MissionStatus");
 		
-		if(MissionStatus == 4) then
+		if(MissionStatus >= 0 and MissionStatus <= 4) then

```

Comment:
[before] gatehouse(0)→Galaihaurat(2) →Equesobillot(3)→Deaufrain(4)→ Vicorpasse

[after]
In retail, we can directly talk to Vicorpasse to proceed after gatehouse(0→Vicropasse)
Also we can do:
gatehouse(0)→Galaihaurat(2)→Vicorpasse 
gatehouse(0)→Galaihaurat(2)→Equesobillot(3)→Vicorpasse
gatehouse(0)→Deaufrain(4)→Vicorpasse
....
2^3=8 kinds of path and that's why checking a range :)